### PR TITLE
Add 'step' to float field (partial view)

### PR DESCRIPTION
**Pull request by lucascaton**

### CODE REVIEW

File: app/views/wallaby/resources/form/_float.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following params are the variables that can be used in this partial.
   <%= form.label field_name, metadata[:label] %>
   <div class="row">
     <div class="col-xs-6 col-sm-4">
-      <%= form.number_field field_name, class: 'form-control' %>
+      <%= form.number_field field_name, class: 'form-control', step: metadata[:step] || 0.1 %>
```
Comment (tian-im) [inline app/views/wallaby/resources/form/_float.html.erb]: passing in an option will be better i think

```
<%= form.number_field field_name, class: 'form-control', metadata[:options] || { step: 0.1 } %>
```

**Review by tian-im — CHANGES_REQUESTED** [review]

File: app/views/wallaby/resources/form/_float.html.erb (hunk 1 above)
Comment (lucascaton) [reply app/views/wallaby/resources/form/_float.html.erb]: @tian-im updated :)

**Comment by tian-im** [comment]
possible to update the spec as well?

**Comment by lucascaton** [comment]
@tian-im sure, will do it soon.

**Comment by lucascaton** [comment]
hey @tian-im - this is the error:

```ruby
SyntaxError:
  /home/travis/build/reinteractive/wallaby/app/views/wallaby/resources/form/_float.html.erb:15: syntax error, unexpected ')', expecting =>
  ...etch(:options, { step: 0.1 }) );@output_buffer.safe_append='
  ...                               ^
  /home/travis/build/reinteractive/wallaby/app/views/wallaby/resources/form/_float.html.erb:22: syntax error, unexpected keyword_ensure, expecting ')'
  /home/travis/build/reinteractive/wallaby/app/views/wallaby/resources/form/_float.html.erb:24: syntax error, unexpected keyword_end, expecting ')'
```

Any idea how to fix it?

File: Gemfile.lock [hunk 2]
```diff
@@ -173,7 +173,7 @@ GEM
       mini_portile2 (~> 2.3.0)
     orm_adapter (0.5.0)
     parallel (1.12.1)
-    parser (2.5.0.4)
+    parser (2.5.0.5)
```
Comment (lucascaton) [inline Gemfile.lock:176]: version `2.5.0.4` has been yanked

**Review by tian-im — APPROVED** [review]

**Pull request merged**